README.md

```diff
@@ -1,2 +1,98 @@
-# components-dotnet-sdk
-Dapr SDK to compose pluggable components in .Net.
+# Dapr Pluggable Components SDK for .NET
+
+[Dapr Pluggable Components](https://docs.dapr.io/concepts/components-concept/#built-in-and-pluggable-components) are Dapr components that reside outside the Dapr runtime but are dynamically registered with the runtime.
+
+This SDK provides a better interface to create Pluggable Components without worrying about the underlying communication protocols and connection resiliency.
+
+## Prerequisites
+
+- [.NET 6 SDK](https://dotnet.microsoft.com/) or later
+- [Dapr 1.9](https://dapr.io/) or later
+
+## Implementing a Pluggable Component
+
+Create an empty ASP.NET project.
+
+```bash
+> dotnet new web --name <project name>
```

Comment:
nit: does it worth to have our own pluggable component template? 

https://learn.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-project-template

Reply:
My first inclination is to say yes, but then you have to think about the overall work involved to create, publish, and maintain the template package (for us), and then learn about, acquire, and then use the template (for customers), versus the steps involved to just use the base ASP.NET template.   I don't know if it's quite worth it.  If there's a huge pickup of customers creating .NET pluggable components, then probably.  Alternatively, we could go with a repo-based "template", where customer forks a repo that has just the minimal amount of scaffolding.

Reply:
Fair enough